socorro/external/postgresql/graphics_devices.py

```diff
@@ -15,28 +15,32 @@ class GraphicsDevices(PostgreSQLBase):
 
     def get(self, **kwargs):
         filters = [
-            ("vendor_hex", None, "str"),
-            ("adapter_hex", None, "str"),
+            ("vendor_hex", None, ["list", "str"]),
+            ("adapter_hex", None, ["list", "str"]),
         ]
         params = external_common.parse_arguments(filters, kwargs)
         for key in ('vendor_hex', 'adapter_hex'):
-            if not params[key]:
+            param = params[key]
+            if not param or (len(param) == 1 and not param[0]):
```

Comment:
What is this?! Something like `[0]` or `['']`? Why is that ever an input to this? Also why is it a problem? Why not just let the query go ahead and if nothing comes back, nothing comes back. 
And why specifically 1? What if there are others. Like `['hex1', 'hex2', '']`